admin/services/data-access/pupil-restart.data.service.js

```diff
@@ -149,8 +209,8 @@ pupilRestartDataService.sqlMarkRestartAsDeleted = async (pupilId, userId) => {
     }
   ]
   const sql = `UPDATE ${sqlService.adminSchema}.[pupilRestart] 
-  SET isDeleted=1, deletedByUser_id=@userId  
-  WHERE [pupil_id]=@pupilId`
+  SET isDeleted=1, deletedByUser_id=@deletedByUser_id  
+  WHERE pupil_id = @pupil_id`
```

Comment:
the parameter names don't _need_ the underscore

Reply:
@GuyHarwood renamed both